test/ObjectField_test.js

```diff
@@ -50,7 +50,10 @@ describe("ObjectField", () => {
     });
 
     it("should render a customized title", () => {
-      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+      const CustomTitleField = ({title, required}) => {
+        const legend = required ? title + '*' : title;
+        return <div id="custom">{legend}</div>;
+      };
```

Comment:
OK, should we still leave the required to the props so that people reading the tests know that they can expect it or just leave it out, because it is not used by the function? Should I revert the README example too (same code)?